### ISSUE TITLE
Add inputValidation.foreground as a theme token

### ISSUE THREAD

**Issue by misolori** [Microsoft/vscode#57536]
In some instances, the `foreground` color does not work well with the `inputValidation` backgrounds. Here's an example where a theme has set `foreground` to `#888`:

![image](https://user-images.githubusercontent.com/35271042/44808471-2f730c80-ab81-11e8-9a61-11fb589da7ff.png)

This forces the theme author to choose only lighter colors that work with `#888`.

If we could set `inputValidation.foreground` to `#FFF` then that would work best with those backgrounds:

![image](https://user-images.githubusercontent.com/35271042/44808534-5df0e780-ab81-11e8-8a0b-52d885a9f050.png)

**Comment by bpasero** [comment]
Yeah having this makes sense to me. I would accept a reasonably small PR if someone wants to chime in. There seems to be quite a bit of places where the color is used.

**Comment by ParkourKarthik** [comment]
I'll pick this up and provide a PR.

**Comment by bpasero** [comment]
Verify: new colors
* `inputValidation.infoForeground`: Input validation foreground color for information severity.
* `inputValidation.warningForeground`: Input validation foreground color for warning severity.
* `inputValidation.errorForeground`: Input validation foreground color for error severity.
